docs/server/management.md

```diff
@@ -0,0 +1,19 @@
+# Work in progress
+
+# Basic Server User Management
+---
+Currently as of v0.19.3 there is no graphical interface for user management from the server level thus you have to use python scripts for user management.
+
+## List all usernames on server  
+```python  
+[ u.name for u in User.select() ]
```

Comment:
I would include the User import here as well.

Reply:
I'm an idiot with python you'll have to help with that one :(

Reply:
the other example already has it! :D 
`from models import User`

Reply:
Oh silly me I'm just copy pasting here from a discord conversation a few days ago in all honesty.  
just that line?

Reply:
ayup!

Reply:
Guess I wasn't super clear, I meant to add the line to the 'list all usernames' code sample as it requires "User" to be imported.

Reply:
Oh, well gosh. I'll go take a quick break for some lunch and I'll get back on it